schemas/groups/notifications.json

```diff
@@ -4,14 +4,19 @@
   "id": "https://signalk.github.io/specification/schemas/groups/notifications.json#",
   "title": "notifications",
   "description": "Notifications, their state, and actions. The notification limits are set in any Signal K key.meta.zones array.",
-
+  "dependencies": {
+  	"message": ["state"]
```

Comment:
Because the depth of the hierarchy is arbitrary I couldnt use `required` as it then applies to all levels of the path. By using `dependencies` we force the use of `state` whereever `message` is used, which is the best compromise I could find.